talk/owt/sdk/p2p/p2pclient.cc

```diff
@@ -347,6 +347,10 @@ void P2PClient::OnMessageReceived(const std::string& remote_id,
                          &P2PClientObserver::OnMessageReceived,
                          remote_id, message);
 }
+void P2PClient::OnStopped(const std::string& remote_id) {
+  removed_pc_channels_.push_back(pc_channels_[remote_id]);
```

Comment:
Please associate the vector push_back/erase with lock.

Reply:
Done.